azext_iot/tests/helpers.py

```diff
@@ -156,6 +159,127 @@ def get_agent_public_ip():
     return requests.head("https://www.wikipedia.org").headers["X-Client-IP"]
 
 
+def get_role_assignments(
+    scope: str,
+    assignee: str = None,
+    role: str = None,
+) -> json:
+    """
+    Get rbac permissions of resource.
+    """
+    role_flag = ""
+    assignee_flag = ""
+
+    if role:
+        role_flag = '--role "{}"'.format(role)
+
+    if assignee:
+        assignee_flag = '--assignee "{}"'.format(assignee)
+
+    return cli.invoke(
+        f'role assignment list --scope "{scope}" {role_flag} {assignee_flag}'
+    ).as_json()
+
+
+def assign_role_assignment(
+    role: str,
+    scope: str,
+    assignee: str,
+    max_tries=10,
+    wait=10
```

Comment:
wait should prob have a default of 60 here since it seems like a decent amount of calls use that as the wait time. Then you can prob remove some wait=60 in calls

Reply:
ok thanks! will take a look!